internal/jobs/tiktok_transcription.go

```diff
@@ -39,9 +36,14 @@ type TikTokTranscriber struct {
 	httpClient    *http.Client
 }
 
-// GetCapabilities returns the capabilities supported by the TikTok transcriber
-func (t *TikTokTranscriber) GetCapabilities() []types.Capability {
-	return []types.Capability{"tiktok-transcription"}
+// GetStructuredCapabilities returns the structured capabilities supported by the TikTok transcriber
+func (t *TikTokTranscriber) GetStructuredCapabilities() []teetypes.JobCapability {
+	return []teetypes.JobCapability{
+		{
+			JobType:      string(teetypes.TiktokJob),
```

Comment:
Ditto. I'll shut up here about this, just check everywhere that we're doing a cast of a structured type, that should never be necessary except when talking to external libraries.